src/Sylius/Behat/Context/Ui/Shop/ProductContext.php

```diff
@@ -310,4 +319,79 @@ public function theFirstProductOnTheListShouldHaveName($name)
             sprintf('Expected first product\'s name to be "%s", but it is "%s".', $name, $actualName)
         );
     }
+
+    /**
+     * @Then I should see :count product reviews
+     */
+    public function iShouldSeeProductReviews($count)
+    {
+        Assert::same(
+            (int) $count,
+            $this->showPage->countReviews(),
+            'Product has %2$s reviews, but should have %s.'
+        );
+    }
+
+    /**
+     * @Then I should see reviews titled :firstReview, :secondReview and :thirdReview
+     */
+    public function iShouldSeeReviewsTitled($firstReview, $secondReview, $thirdReview)
```

Comment:
Since VariadicsExtension is enabled, we could roll up all this parameters to one `...$reviews` and then use foreach to iterate over them. The biggest benefit of such a solution would be if anybody would like to assert one, two or more then three reviews titles it would be enough to add step definition